src/versions.rs

```diff
@@ -42,6 +42,84 @@ pub struct Crate {
     pub publish: Option<bool>,
 }
 
+pub async fn get_orml_crates_and_version(
+    base_url: &str,
+    version: &str,
+) -> Result<Option<(Vec<String>, String)>, Box<dyn std::error::Error>> {
+    if get_release_branches_versions(Repository::Orml)
+        .await?
+        .contains(&version.to_string())
+    {
+        let version_url = format!(
+            "{}/open-web3-stack/open-runtime-module-library/polkadot-v{}/Cargo.dev.toml",
+            base_url, version
+        );
+        let response = reqwest::Client::new()
+            .get(&version_url)
+            .header("User-Agent", "reqwest")
+            .header("Accept", "application/vnd.github.v3+json")
+            .send()
+            .await?;
+
+        let content = response.text().await?;
+
+        Ok(Some(parse_orml_workspace_members(&content)))
+    } else {
+        log::error!(
+            "No matching ORML release version found for corresponding polkadot-sdk version."
```

Comment:
Somehow this just errors for me `psvm -v 1.0.0 --orml -c `:

```pre
Failed to get Plan.toml, falling back to Cargo.lock.
Error: Error { inner: Error { inner: TomlError { message: "expected `.`, `=`", raw: Some("404: Not Found"), keys: [], span: Some(3..4) } } }
```
any idea?

Reply:
v1.0.0 is not a release version for polkadot-sdk so it isn't available. Can try listing the available versions using the `--list` flag

Reply:
Okay. We should improve the error messages here 😆 (but can be done afterwards).

Reply:
Yeah I was super confused while working on the `--check` flag when I was hit by this 😆. Now I know what it means.